packages/test/test-version-utils/src/itSkipsOnFailure.ts

```diff
@@ -50,7 +56,9 @@ export type SkippedErrorExpectingTestWithDriverType = (
 
 /**
  * Similar to mocha's it function, but allow skipping for some if the error
```

Comment:
Nit (while we're here)
```suggestion
 * Similar to mocha's `it` function, but allow skipping for some if the error
```